install_sct

```diff
@@ -591,11 +591,21 @@ osx)
 esac
 
 # Run conda installer
+print info "Installing Miniconda..."
 run bash "$TMP_DIR/miniconda.sh" -p "$SCT_DIR/$PYTHON_DIR" -b -f
 
+# Workaround for WSL "HTTP 000 Connection Failed" error
+# See: https://github.com/conda/conda/issues/9948#issuecomment-909989810
+# Syntax explanation: https://stackoverflow.com/a/6085237
+find "$SCT_DIR/$PYTHON_DIR" -type f -exec touch {} +
+
 # create conda environment
+print info "Creating conda environment..."
 python/bin/conda create -y -n venv_sct python=3.7
 
+# Reapply the touch fix to avoid connection issues on WSL1
+find "$SCT_DIR/$PYTHON_DIR" -type f -exec touch {} +
+
```

Comment:
Based on discussion in our last SCT Course meeting, I think the least invasive method would be to split the "touch" lines into a separate PR, and only backport these changes. 

(That's because the rest of the fixes are for the CI runners (not necessary to backport, since our CI runs off `master`) and documentation (not necessary to backport, since our docs are pointed at `master`).)

That would let us keep the changes to `5.4` to a minimum.

Reply:
5.4 has been updated to include the fix from #3574.

See:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/commits/5.4